runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -4623,10 +4648,9 @@ void init_amHandling(void) {
   // least one is running.
   //
   atomic_init_bool(&amHandlersExit, false);
-
   PTHREAD_CHK(pthread_mutex_lock(&amStartStopMutex));
   for (int i = 0; i < numAmHandlers; i++) {
-    CHK_TRUE(chpl_task_createCommTask(amHandler, NULL) == 0);
+    CHK_TRUE(chpl_task_createCommTask(amHandler, &reservedCPUs[i]) == 0);
```

Comment:
What do you think about `chpl_task_createCommTask` doing the pinning instead of having each comm layer doing the pinning?

Reply:
Seems reasonable, let me look into it.